interface/wx/pen.h

```diff
@@ -37,10 +37,12 @@ enum wxPenStyle
         /**< No pen is used. */
 
     wxPENSTYLE_STIPPLE_MASK_OPAQUE,
-        /**< @todo WHAT's this? */
+        /**< Uses a bitmap as a stipple; the mask is used for blitting
+             monochrome using text foreground and background colors. */
```

Comment:
Is there something missing after "monochrome"? I have trouble parsing this sentence in the current state.

Reply:
I copied this description from the sibling `wxBRUSHSTYLE_STIPPLE_MASK_OPAQUE` enum value.  I assumed "monochrome" was used as an adverb here and I couldn't wordsmith it any differently...so I just left it as-is.

Reply:
The existing description doesn't make any sense to me neither :-(

I also don't think this is actually supported under any platform, so perhaps it would be better to just remove its documentation completely.

Reply:
Should I remove it entirely from the interface file then?

Reply:
I think it would be best, unless I'm missing something -- but a quick grep doesn't seem to find support for this in any port. But please double check it.